styles/index.css

```diff
@@ -0,0 +1,31 @@
+header, footer {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    overflow: hidden;
+    background-color:#4ABDAC; 
+    height: 350%;
+    width: 100%;
+    justify-content: flex-start;
+    align-items: center;
+}
```

Comment:
Your `nav` element is inside the `header` and the way it's styled keeps your navigation components from following the flexbox rules.

You might prefer what happens if you update your rules to rely on flexbox and styling the banner in the `nav` tag instead of adding spacing with the `header` or the `a` tags' rules:

```css
.top-banner {
    display: flex;
    justify-content: space-evenly;
    width: 100%;
    padding: 2rem;
}

header {
    height: unset;
}

.top-banner a {
    margin: unset;
}
```